### DISCUSSION TITLE
New cockpit updates not being found on Rocky Linux/CentOS 8

### DISCUSSION THREAD

**Discussion by wsandy1** [cockpit-project/cockpit#17061] [Q&A] [answered]
I'm trying to update my cockpit version to release 264. I believe it should be on the stream by now, but even if it isn't, I know new updates aren't being found as apparently cockpit is still on version 251.3:
![image](https://user-images.githubusercontent.com/62423608/155963985-ea8e163a-98e4-4236-9a1b-c9c2ef1c65eb.png)

If I run `yum clean all` followed by `yum update cockpit`, it says no updates are found:
![image](https://user-images.githubusercontent.com/62423608/155964124-cb17af37-6674-440e-ae17-e0d37cf7d556.png)

Fairly sure there should be versions newer than 251 on the stream, but I've got no idea how to convince rocky to find them. Cheers.

**Comment by martinpitt** [comment, marked as answer]
I'm not familiar with Rocky Linux, but I think it made a specific point about *not* following CentOS 8 stream, but basing off the latest stable "legacy" CentOS based on RHEL 8.5. So you won't find updates there. (YAFIYGI!)

If you *do* want package updates, move to CentOS 8 stream, or use our [COPR](https://copr.fedorainfracloud.org/coprs/g/cockpit/cockpit-preview/) for the latest cockpit versions.

**Reply by wsandy1** [reply]
COPR works great cheers

**Reply by fmsy** [reply]
As of today COPR no longer works as it is missing an EPEL8 build. @martinpitt 

It was present until at least 22/04/05 @ https://download.copr.fedorainfracloud.org/results/@cockpit/cockpit-preview/epel-8-x86_64/

![image](https://user-images.githubusercontent.com/3216591/171323969-ce2ebbec-893c-4852-a836-a813e0dd9b6c.png)

Are there any plans on providing EPEL8 builds again ?

Using the centos8stream builds is not an options as they depend on non-existent versions of policy-targeted

![image](https://user-images.githubusercontent.com/3216591/171324610-e53c204c-ad5b-4129-9e28-31a843cc2a37.png)

**Reply by fmsy** [reply]
concerning **YAFIYGI** : the resulting need to publish an additional build for latest packages from projects like cockpit has been caused by the RedHat "Centos Stream"-Change, which leads to my holy oppinion that, if early feedback is one of the reasons for the cockpit-team to provide the COPR preview, then this extra build-target is what RedHat has asked for, not the users!
